File: fstmerge/examples/Stack_python/Flushable/stack.py
class Stack:
	items = xlist()
	
	def push(self, item):
		self.items.append(item)
	
	def pop(self):
		if len(self.items) > 0:
			return self.items.pop()
		else:
			return None

	def flush(self):
		pass

File: fstmerge/examples/Stack/Flushable/Stack.java
import java.util.LinkedList;
import java.io.Flushable;

public class Stack<T> implements Flushable {
	private LinkedList<T> items = new LinkedList<T>();
	public void push(T item) {
		items.addFirst(item);
	}
	public T pop() {
		if(items.size() > 0)
			return items.removeFirst();
		else
			return null;
	}
	public void flush() { }
}